docs/GroupDefs.hpp

```diff
@@ -48,6 +48,42 @@
 /*!
  * \defgroup ControlSystemGroup Control System
  * \brief Contains control system elements
+ *
+ * The control system manages the time-dependent mapping between the fixed
+ * computational frame (grid frame) and the inertial frame. The time-dependent
+ * parameters of the mapping are adjusted by a feedback control system in order
+ * to follow the dynamical evolution of the black holes. In BBH simulations the
+ * map is typically a composition of maps that include translation, rotation,
+ * scaling, shape, etc. Each map under the governance of the control system has
+ * an associated time-dependent map parameter \f$\lambda(t)\f$ that is regularly
+ * updated by a control signal \f$U(t)\f$, which is determined by an error
+ * function \f$Q(t)\f$ that is uniquely specified for each map.
+ *
+ * The key components of the control system are:
+ * - FunctionsOfTime: each map has an associated FunctionOfTime which holds the
+ *   map parameter \f$\lambda(t)\f$ and relevant time derivatives
+ * - MeasureControl: each map has an associated MeasureControl which computes
+ *   the error, \f$Q(t)\f$, in the mapping
+ * - Averager: an averager can be used to average out the noise in the 'raw'
+ *   \f$Q(t)\f$ returned by the MeasureControl
+ * - Controller: the map controller computes the control signal \f$U(t)\f$ from
```

Comment:
It is not clear from the documentation what is done with the control signal U(t).  (e.g. is it added to lambda? Is it subtracted from lambda? ) Say above somewhere that lambda is piecewise constant, and the Nth deriv of lambda is set to U(t) at regular intervals.